internal/release/postinstall.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+# Copyright The OpenTelemetry Authors
+# Copyright Dynatrace LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if command -v systemctl >/dev/null 2>&1; then
+    systemctl enable dynatrace-otel-collector.service
+    if [ -f /etc/dynatrace-otel-collector/config.yaml ]; then
+        systemctl start dynatrace-otel-collector.service
+    fi
```

Comment:
should we log here if no config is available?

Reply:
Something like "collector installed, but no config found, skipping startup"?

Reply:
That would only show up right after installation though. But I can add something.

Reply:
I think that's okay. I know that many things automatically start up when you install them, so people might be expecting it to start up right away. Having a note that this one doesn't automatically start would make that very clear.